modules/configuration/ajax/process.php

```diff
@@ -89,7 +89,7 @@
                 continue;
             }
             if (isDuplicate($ConfigSettingsID, $value)) {
-                displayError(400, "Duplicate Value In The Instrument Form");
+                displayError(400, "Duplicate value submitted");
```

Comment:
Would there be benefit by showing the `$value` in the dissplayError?

Reply:
Probably. I'll do this